src/server/pbsd_init.c

```diff
@@ -1449,6 +1452,54 @@ pbsd_init(int type)
 #endif
 
 	hook_track_recov();
+	
```

Comment:
Some comment here about what the following code block intends to do would be very helpful for posterity